### DISCUSSION TITLE
Will my Free Page Expire?

### DISCUSSION THREAD

**Discussion by mitosiscalculator** [community/community#60633] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello! Total newbie here: if I've made a simple free Github website, will the website ever expire? If so is it contingent on use, a certain time period, etc.?

**Comment by jussihi** [comment, marked as answer]
There is no rules for site expiration, for usage limits see [Github's official usage limits for GH websites](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits). **However, it may be any day that GitHub decides to change their policy.** But with current policy there is no expiration rules for Github Pages.

**Reply by GhostOf0days** [reply]
There is no expiration. However, OP should use Netlify, Heroku, or Vercel instead. Netlify for simple use cases/websites and Vercel for larger, more complex use cases/websites.
